.github/workflows/build.yaml

```
@@ -0,0 +1,181 @@
+        type: choice
+        options:
+          - studio-services
+        default: studio-services
+      service:
+        description: 'Name of the service to build and deploy'
+        required: true
+        type: choice
+        options:
+          - public-service
+          - studio-pdf-helper-service
+        default: "audit-service"
```

actionlint (1.7.7)

default value "audit-service" of "service" input is not included in its options "\"public-service\", \"studio-pdf-helper-service\""

(events)